Makefile

```diff
@@ -0,0 +1,10 @@
+cs:
+	vendor/squizlabs/php_codesniffer/bin/phpcs .
+
+phpmd:
+	vendor/phpmd/phpmd/src/bin/phpmd . text phpmd.xml --exclude vendor/
+
+phpunit:
+	vendor/phpunit/phpunit/phpunit --configuration phpunit.xml.dist tests/
+
```

Comment:
For being consistent across projects, please add the `covers` target (and add the `covers-validator` tool) and create the `test` and `ci` targets like in the FundraisingFronted. Then use the `make ci` target in the `.travis.yml` file.

Reply:
As outlined in the daily: `covers` cannot (easily) be added due to a version conflict with Piwik on the `symfony/console` dependency.